spec/factories/repository_factory.rb

```diff
@@ -43,5 +43,14 @@
       remote_type { 'mirror' }
       source_type { 'git' }
     end
+
+    factory :private_repository do |repository|
+      repository.after(:build) do |repository|
+        repository.access = 'private_rw'
```

Comment:
Please move the access up a line (this is a model attribute which can be set directly):
```ruby
    factory :private_repository do |repository|
      access { 'private_rw' }
      repository.after(:build) do |repository|
        editor = create :user
        ```